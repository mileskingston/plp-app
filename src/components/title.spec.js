import React from 'react';
import renderer from 'react-test-renderer';
import Title from './title';

describe('Title Component', () => {
  const props = {
    heading: 'Title',
    totalItems: 3,
  };

  it('renders component', () => {
    let component = renderer.create(<Title {...props} />).toJSON();
    expect(component).toMatchSnapshot();
  });

  it('renders component with no items label', () => {
    let component = renderer.create(<Title {...props} totalItems={0} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});