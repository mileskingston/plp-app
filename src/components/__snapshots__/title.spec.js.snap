// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component renders component 1`] = `
<h1
  className="title"
>
  Title
   
  <span
    className="title__total"
  >
    3 Items
  </span>
</h1>
`;

exports[`Title Component renders component with no items label 1`] = `
<h1
  className="title"
>
  Title
   
</h1>
`;
